src/less/components.less

```diff
@@ -42,5 +42,4 @@
 @import 'components/table/index';
 @import 'components/tabset/index';
 @import 'components/text-input/index';
-@import 'components/textarea/index';
-@import 'components/toast/index';
+@import 'components/textarea/index';
```

Comment:
Please add a newline after this statement.

Reply:
Please check the changes in latest commit.